docs/topics/articles.md

```diff
@@ -0,0 +1,184 @@
+# `Article`s and `ListingPage`s
+
+`Article`s are publishable content types that live in collections. Normally
+articles are mounted under a parent's URL.
+
+An example of an article is a Press Release, which would be mounted under a
+Press Release Listing Page. The Listing Page has the URL "/press" and thus all
+the Press Releases have the URL "/press/<slug>".
+
+## `ArticleBase`
+
+The `icekit.articles.models.ArticleBase` inherits `title`, `slug` and
+publishing fields, and each `ArticleBase` subclass implements some key
+functionality:
+
+1. **Links to parents.**
+In order for an Article to know its URL (which is based on a parent's URL), it
+should define a `parent` link, which is normally a link to a `ListingPage`.
```

Comment:
What's wrong with the way it is done in SFMOMA? If we want multiple list pages, one for each category, all at top level URLs, just add a category field to the list page plugin and filter the objects being listed by category. If the category list pages can all be under a common URL, then include the category as a URL parameter. I think this is by far the more common case and least confusing or error prone solution to filtered list pages (vs. FKs from each object being listed to a specific list page instance). Having an FK field that is ignored (at best) or faked when not needed (which is probably most of the time) in order to support a less common / more complex case seems like a bad trade-off. And I'm still worried about cascade deletes. The other `on_delete` behaviours (protect, set null) might not be much better either.